src/current/v24.1/cutover-replication.md

```diff
@@ -154,16 +154,115 @@ At this point, the primary and standby clusters are entirely independent. You wi
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER vc_a STOP SERVICE;
+    ~~~
+
+1. Open another terminal window and connect to the system virtual cluster for **cluster B**:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --url \
+    "postgresql://{user}@{node IP or hostname cluster B}:26257?options=-ccluster=system&sslmode=verify-full" \
+    --certs-dir "certs"
+    ~~~
+
+1. From the system virtual cluster on **cluster B**, enable rangefeeds:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING kv.rangefeed.enabled = 'true';
+    ~~~
+
+1. From the system virtual cluster on **cluster A**, start the replication from cluster B to cluster A:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER {vc_a} START REPLICATION OF {vc_b} ON 'postgresql://{user}@{ node IP or hostname cluster B}:26257?options=-ccluster=system&sslmode=verify-full&sslrootcert=certs/{standby cert}.crt';
+    ~~~
+
+    This will rewind the virtual cluster on **A** back to the time at which the same virtual cluster on **B** diverged from it. **Cluster A** will check with **cluster B** to confirm that its virtual cluster was replicated from **cluster A** as part of the original [physical cluster replication stream]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
+
+    {{site.data.alerts.callout_success}}
+    For details on connection strings, refer to the [Connection reference]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#connection-reference).
+    {{site.data.alerts.end}}
+
+1. Check the status of the virtual cluster on **A**:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTER {vc_a};
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+     id |  name  |     data_state     | service_mode
+    ----+--------+--------------------+---------------
+      1 | system | ready              | shared
+      3 | {vc_a} | replicating        | none
+      4 | test   | replicating        | none
+      (2 rows)
+    ~~~
+
+1. Run the following from **cluster A** to start the cutover:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER {vc_a} COMPLETE REPLICATION TO LATEST;
+    ~~~
+
+    The `cutover_time` is the timestamp at which the replicated data is consistent. The cluster will revert any data above this timestamp:
+
+    ~~~
+               cutover_time
+    ----------------------------------
+      1714497890000000000.0000000000
+    (1 row)
+    ~~~
+
+1. From **cluster A**, bring the virtual cluster online with:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER {vc_a} START SERVICE SHARED;
+    ~~~
+
+1. To make **cluster A's** virtual cluster the default for connection strings, set the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}):
```

Comment:
Maybe link to https://www.cockroachlabs.com/docs/stable/work-with-virtual-clusters#sql-clients?